### PR TITLE
Add provisioning codes of multiple CDK stacks

### PR DESCRIPTION
**Pull request by tbrand**
~~Please do NOT merge this patch since it just shows sample codes.~~

### CODE REVIEW

File: app/app.py [hunk 1]
```diff
@@ -8,10 +8,7 @@
 import uuid
 
 
-dynamodb = boto3.resource(
-    'dynamodb',
-    endpoint_url=os.environ['DYNAMODB_ENDPOINT_URL'],
```
Comment (studio3104) [inline app/app.py]: We cannot remove this line, need to specify it to run locally.
I'm gonna make it back later.

File: app/app.py [hunk 2]
```diff
@@ -102,7 +99,7 @@ def patch(cls, event: EventType) -> Tuple[int, Dict[str, str]]:
 
 
 def dispatch_request(event: EventType, context: ContextType) -> ResponseType:
-    request_path = event['requestContext']['path']
+    request_path = event['requestContext']['resourcePath']
```
Comment (studio3104) [inline app/app.py]: Good point 😸
But we've gotta modify unit tests as well, I'm gonna update them later.

File: app/app.py (hunk 1 above)
Comment (tbrand) [reply app/app.py]: Oh I see. I totally forgot about testing!

File: app/app.py (hunk 2 above)
Comment (studio3104) [reply app/app.py]: https://github.com/aws-samples/aws-rapid-prototyping-with-python/pull/1/commits/d3909e9ec7d3847efadc89b2d5696ae4c87f310a

**Pull request merged**